entity-service-impl/src/main/java/org/hypertrace/entity/query/service/EntityQueryServiceImpl.java

```diff
@@ -550,6 +555,11 @@ private List<Document> getProjectedDocuments(
                     IdentifierExpression.of(EntityServiceConstants.ENTITY_ID),
                     IN,
                     ConstantExpression.ofStrings(entityIdList)))
+            .expression(
```

Comment:
Does filter accept multiple expressions and implicitly and them or is this just overwriting the prior filter?